ray-operator/controllers/ray/common/pod.go

```diff
@@ -675,8 +675,9 @@ func concatenateContainerCommand(nodeType rayiov1alpha1.RayNodeType, rayStartPar
 
 func convertParamMap(rayStartParams map[string]string) (s string) {
 	flags := new(bytes.Buffer)
+	nonFlagParams := []string{"log-color", "include-dashboard"}
 	for k, v := range rayStartParams {
-		if strings.ToLower(v) == "true" {
+		if strings.ToLower(v) == "true" && !utils.Contains(nonFlagParams, k) {
```

Comment:
nice catch. I think `strings.ToLower(v) == "true"` is not always correct for such cases. We internally use `--dashboard-host` directly and didn't notice this issue.  The change looks good to me, we may leave an eye on new added flags in the future